pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -14,6 +14,60 @@ import (
 // HyperConvergedName is the name of the HyperConverged resource that will be reconciled
 const HyperConvergedName = "kubevirt-hyperconverged"
 
+// NodePlacement describes node scheduling configuration.
+// nodeSelector and tolerations can be independently configured and
+// are going to be propagated by HCO to all the managed operators.
+type NodePlacement struct {
+	// nodeSelector is the node selector applied to the relevant kind of pods
+	// It specifies a map of key-value pairs: for the pod to be eligible to run on a node,
+	// the node must have each of the indicated key-value pairs as labels
+	// (it can have additional labels as well).
+	// See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// affinity enables pod affinity/anti-affinity placement expanding the types of constraints
+	// that can be expressed with nodeSelector.
+	// affinity is going to be applied to the relevant kind of pods in parallel with nodeSelector
+	// See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+	// +optional
+	Affinity corev1.Affinity `json:"affinity,omitempty"`
+
+	// tolerations is a list of tolerations applied to the relevant kind of pods
+	// See https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ for more info.
+	// These are additional tolerations other than default ones.
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+}
+
+// HyperConvergedPlacement is the schema for pod placement configuration that is going to
+// be propagated by HCO to its managed operators.
+type HyperConvergedPlacement struct {
+	// infra NodePlacement influences the placement (nodeSelector and tolerations)
```

Comment:
Maybe omit the detail (nodeSelector and tolerations) or add the word "affinity" to it?

Reply:
yes, thanks